spec/author_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './spec_helper'
+
+describe Author do
+  context 'Testing Author' do
+    before(:each) do
+      @author = Author.new('Anthony', 'Mwenyo')
+    end
+    it 'Should be instance of the Author class' do
+      expect(@author).to be_instance_of Author
+    end
+    it 'Should have the firstname equal to Anthony' do
+      expect(@author.firstname).to eq 'Anthony'
+    end
+    item = Item.new('2022-08-03')
```

Comment:
Don't you think moving this variable to the bottom test case would be better? this place doesn't look perfect for defining a variable.